sdk/containerregistry/service.projects

```diff
@@ -7,12 +7,16 @@
   the set of projects to match what type of SDK we are interested in building.
 
   SDKType values
-   - empty or "client" will build the new track 2 client/data libraries
+   - "all" or "client" will build the new track 2 client/data libraries
```

Comment:
May make more sense to add a new line above saying that empty or all will build all of the things and leave just "client" for the second line.

Reply:
Good point.